site/multiDim/MultiDimDataPageContent.tsx

```diff
@@ -261,6 +261,7 @@ export const MultiDimDataPageContent = ({
 
     const grapherConfigComputed = useMemo(() => {
         const baseConfig: GrapherProgrammaticInterface = {
+            bindUrlToWindow: true,
```

Comment:
Is there any particular reason why you've made this change?
I haven't really noticed a difference before and after, and I feel like I've avoided adding this property in the past because it wouldn't add mdim-specific query params like `?povertyLine=`.

The one thing I've noticed not working is the `?tab=` query param, for whatever reason.
But then, in fact, it hasn't worked correctly beforehand and is also not working correctly now, so there's no change there :)

Reply:
I've made it to synchronize the window title with the grapher title. Without it, the window title is just "Our World in Data". This change makes it similar to normal data pages, e.g. "$40 a day - Share of population in poverty, 1990 to 2019".

> it wouldn't add mdim-specific query params like `?povertyLine=`

I'm not sure what do you mean by this. Can you expand on it?

> The one thing I've noticed not working is the `?tab=` query param

I'll track and hopefully also fix this separately.

Reply:
Ah, I see!

What I mean is that [these lines of code in `bindToWindow()`](https://github.com/owid/owid-grapher/blob/acc3c3e4a66bc80479c628aba54136d4ebf953d3/packages/%40ourworldindata/grapher/src/core/Grapher.tsx#L2934-L2936) - which is the main effect of `bindUrlToWindow` - make it so that it is grapher's responsibility to update query params.
But grapher doesn't know about any mdim-specific query params, like `?povertyLine`. Which means that grapher will update the query params to something like `?time=2011`, only for the Mdim code to update it to `?time=2011&povertyLine=all`.

You can see this in the browser history, for example - there are two history entries for each chart change: one without and one with the mdim query params.

Reply:
Ah, I somehow missed this. I agree we should change it, however when I change some mdim settings and the URL updates, `history.length` stays 1 no matter how many times I make the change. Did you mean some other history?

Reply:
I meant the actual, user-facing browser history in this case, e.g. chrome://history.

Different browsers might behave differently, but in Chrome & Edge I see duplicate history entries for each change.